packages/leancode_cubit_utils/README.md

```diff
@@ -19,14 +19,46 @@ Implementation of cubits for handling [CQRS](https://pub.dev/packages/cqrs) quer
 
 ## Single Request Utils
 
+### `handleResult`
+
+`handleResult` is a client-specific method needed for handling the API response. It can be defined once and used repeatedly as a mixin.
```

Comment:
We should also mention that it can be used in the base class for our cubits (we can link to the leancode_cubit_utils_cqrs implementation as an example). Currently, it is a little confusing. The examples in this readme suggest using mixins everywhere, but the CQRS implementation uses this handler internally. It is more convenient in projects to create your own classes AppRequestCubit, AppPrerequest etc. and forget about this handleResult method.

Reply:
+1 run method in prerequest

Reply:
I think I've changed it properly but please check on your own